astropy/convolution/core.py

```diff
@@ -180,7 +180,7 @@ def __array__(self):
         """
         return self._array
 
-    def __array_wrap__(self, array, context=None):
+    def __array_wrap__(self, array, context=None, return_scalar=False):
```

Comment:
Looks like this was added by @adonath many years ago to fix something:

* https://github.com/astropy/astropy/pull/2652

Reply:
It looks like `__array_wrap__` was added to allow a `Kernel` object to be multiplied by a scalar.  Is there a better way to do that?

Reply:
Ah, yes, that makes sense. Two solutions: (1) set `__array_priority__ = 1000`, so that you get precedence, (2) set `__array_ufunc__ = None` to indicate numpy should not try to do any arithmetic. The latter is the more "modern" one and probably best here - one can always revisit if it is decided to allow more than the basic operations. But obviously check whether the tests still pass!

Reply:
let's try `__array_ufunc__ = None`